Dockerfile

```diff
@@ -1,18 +1,56 @@
-FROM python:3.6
+### Build python virtualenv
+FROM python:3.6-stretch as build-python
 ENV PYTHONUNBUFFERED 1
 
 RUN \
   apt-get -y update && \
-  apt-get install -y gettext && \
-  apt-get clean
+  apt-get install -y gettext python3-virtualenv virtualenv && \
```

Comment:
Why installing both `python3-virtualenv` and `virtualenv`? `virtualenv` alone should be enough (which depends on 	`python3-virtualenv`).

Reply:
This was required in jessie, you are right that it should not be needed now